supporting_content/dimensions/usageunit.md

```diff
@@ -0,0 +1,20 @@
+# Usage Unit
+
+## Example provider mappings 
+
+Current column mappings found in available data sets:
```

Comment:
Should we point out that some of the currently available columns actually represent close match (pricing units and not usage units) or do we assume that the values are/will be identical?

Reply:
Discussed on VS3 call.  @ijurica could you clarify on what belongs here?  We weren't quite sure.

Reply:
Only GCP provides two columns: `usage.unit`, which corresponds to UsageUnit, and `usage.pricing_unit`, which corresponds to PricingUnit. 
IMO, in case of AWS and Azure, those columns are the **closest match to UsageUnit** and an **exact match for PricingUnit**. (we should probably confirm this with Michael and Yuriy)

Reply:
Since the spec is specifying what must be provided in the billing data, I think using `usage.unit` is fine for the 1.0 example. `usage.pricing_unit` looks like it only exists in their pricing API/billing catalog. It is not being natively published to the billing data that I have sifted through, only `usage.unit`.